internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -211,12 +223,26 @@ func ParseEventRecord(e EventRecord) (sdktypes.EventRecord, error) {
 
 type Env struct {
 	EnvID     string `gorm:"primaryKey"`
-	ProjectID string `gorm:"index"`
+	ProjectID string `gorm:"index;foreignKey"`
```

Comment:
do you need the foreign key tag even if you added the project field ?

Reply:
If I remove it gorm will guess it I think, but explicit IMO is better then implicit.